### PR TITLE
LRQA-56378

### PR DESCRIPTION
**Pull request by brookedalton**

### CODE REVIEW

**Comment by brookedalton** [comment]
ci:test:search-es7

**Comment by brookedalton** [comment]
ci:test:search-es7

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/elasticsearch/Elasticsearch7EE.testcase [hunk 1]
```diff
@@ -966,6 +966,93 @@ definition {
 		}
 	}
 
+	test MultipleElasticsearchConnections {
```
Comment (timpak) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/elasticsearch/Elasticsearch7EE.testcase]: Test names should start with a verb unless it's a smoke test.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/elasticsearch/Elasticsearch7EE.testcase [hunk 2]
```diff
@@ -966,6 +966,93 @@ definition {
...
+		property osgi.app.includes = "portal-search-elasticsearch7";
+		property osgi.module.configuration.file.names = "com.liferay.portal.bundle.blacklist.internal.BundleBlacklistConfiguration.config";
+		property osgi.module.configurations = "blacklistBundleSymbolicNames=&quot;com.liferay.portal.mobile.device.detection.fiftyonedegrees.api,com.liferay.portal.mobile.device.detection.fiftyonedegrees.enterprise,com.liferay.portal.mobile.device.detection.fiftyonedegrees.enterprise.test.data,com.liferay.portal.mobile.device.detection.fiftyonedegrees,com.liferay.portal.search.elasticsearch6.impl,com.liferay.portal.search.elasticsearch6.spi,com.liferay.portal.search.elasticsearch6.api,com.liferay.portal.search.elasticsearch6.xpack.security.impl,com.liferay.portal.search.elasticsearch6.xpack.monitoring.web,com.liferay.portal.search.tuning.rankings.web&quot;";
```
Comment (timpak) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/elasticsearch/Elasticsearch7EE.testcase]: The blacklist for CCR includes com.liferay.portal.search.tuning.rankings.web as a workaround. For Multiple Connections, you can rely on the default blacklist and remove these two lines.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/elasticsearch/Elasticsearch7EE.testcase [hunk 3]
```diff
@@ -966,6 +966,93 @@ definition {
...
+		property portal.upstream = "quarantine";
+		property remote.elasticsearch.enabled = "true";
+		property remote.elasticsearch.clusters.enabled = "true";
```
Comment (timpak) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/elasticsearch/Elasticsearch7EE.testcase]: The reindex test for CCR was added after this PR was created. Can you change the property for that test as well and squash it into your rename commit?

**Comment by timpak** [comment]
Hey Brooke, really nice job with the test and logic changes. Just a couple of minor updates needed. Can you address those, rebase, and send another pull?

**Pull request closed without merging**